website/docs/Ganglion/08-Ganglion_Data_Format.md

```diff
@@ -195,259 +105,24 @@ The Ganglion will sometimes send verbose data over the BLE connection. In this c
 | `206`              | `0xCE`               | `multi`     | Part of ASCII message                    |
 | `207`              | `0xCF`               | `multiStop` | End of ASCII message                     |
 
-## Example Interpreting Ganglion Data
```

Comment:
This too.

Reply:
This is example is outdated by the new firmware release. The firmware simply sends the MSBs now and doesn't involve a multi-sample reconstruction strategy.